sdk/eventhub/Azure.Messaging.EventHubs/src/Consumer/EventHubConsumerClient.cs

```diff
@@ -891,6 +899,15 @@ void exceptionCallback(Exception ex)
 
                         if (retryDelay.HasValue)
                         {
+                            // If items were being emitted at the time of the exception, skip to the
+                            // last active item is preserved so that no events are lost and duplicates are
```

Comment:
nit: `skip to the last active item is preserved so...` seems a bit awkward. Did you mix two sentences?

Reply:
Yeah,  I'm guessing that was an artifact of writing it between other tasks.   Fixed.  😄